src/api.js

```diff
@@ -0,0 +1,18 @@
+import axios from "axios";
+
+const api = axios.create({
+  baseURL: "https://kates-nc-news.onrender.com/api",
+});
+
+function getArticles() {
+  return api
+    .get("/articles")
+    .then(({ data }) => {
+      return data.articles;
+    })
+    .catch((err) => {
```

Comment:
You'll likely want to catch any errors in the component so that you can display an error message to the user if something goes wrong. You'll need to remove the catch block here to achieve that.